other/ingress_host_match_tls/ingress-host-match-tls.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     policies.kyverno.io/severity: medium
     kyverno.io/kyverno-version: 1.6.0
     policies.kyverno.io/minversion: 1.6.0
-    kyverno.io/kubernetes-version: "1.23"
+    kyverno.io/kubernetes-version: "1.20"
```

Comment:
This indicates you tested and verified this policy using Kubernetes 1.20. Is that correct?